### PR TITLE
sys-cluster/ipvsadm: add package to coreos

### PR DESCRIPTION
**Pull request by ajeddeloh**
Depends on https://github.com/coreos/portage-stable/pull/575

### CODE REVIEW

File: coreos-base/coreos/coreos-0.0.1.ebuild [hunk 1]
```diff
@@ -164,4 +164,5 @@ RDEPEND="${RDEPEND}
 	sys-libs/timezone-data
 	sys-process/lsof
 	sys-process/procps
+	sys-cluster/ipvsadm
```
Comment (dm0-) [inline coreos-base/coreos/coreos-0.0.1.ebuild]: This list is alphabetical.

**Comment by ajeddeloh** [comment]
Review bump. ipvsadm didn't have the arm64 keyword at all but it appears to build and run.

**Comment by euank** [comment]
wrt the "appears to build and run", can we get a kola test alongside this to codify that?

**Comment by ajeddeloh** [comment]
Fixed build issues. Builds without `libnl` or `ipvsadm` being installed locally in the SDK on both arm64 and amd64.
@euank What were you looking for in a kola test? Just that it runs at all or something more exhaustive?

**Comment by euank** [comment]
@ajeddeloh I think we should get into the habit of adding "smoke test" like kola tests for new binaries.

So yes, verify basic functionality. In this case, just adding, listing, and then deleting a few ipvsadm "services" would be a good minimum check to have.

**Comment by ajeddeloh** [comment]
Added INSTALL_MASK and PKG_INSTALL_MASK for libnl

File: coreos/config/env/dev-libs/libnl [hunk 2]
```diff
@@ -0,0 +1,2 @@
+INSTALL_MASK+=" /etc/libnl/ /etc/libnl/classid /etc/libnl/pktloc"
+PKG_INSTALL_MASK+=" /etc/libnl/ /etc/libnl/classid /etc/libnl/pktloc"
```
Comment (dm0-) [inline coreos/config/env/dev-libs/libnl]: Can you just add `/etc/libnl` to `profiles/coreos/targets/generic/make.defaults`?  It's better to keep these centralized so we don't lose track of them.

**Comment by ajeddeloh** [comment]
Fixed.

File: profiles/coreos/targets/generic/make.defaults [hunk 3]
```diff
@@ -33,6 +33,7 @@ INSTALL_MASK="${INSTALL_MASK}
   /etc/sudoers
   /etc/wgetrc
   /etc/xinetd.d
+  /etc/libnl
```
Comment (dm0-) [inline profiles/coreos/targets/generic/make.defaults]: This list is alphabetical.  (Or it should be ... `lvm` does not go before `logrotate`.)

**Pull request merged**